dirs

```diff
@@ -0,0 +1,8 @@
+STACK_FOLDERS = \
```

Comment:
Do we really need that? Using the Docker.source file as flag is enough, in my opinion.

Reply:
This is used so that we don't have to run find to enumerate all folders with stacks or dockerfiles